crates/node/src/commands/run.rs

```diff
@@ -186,13 +186,21 @@ pub fn run_node(mut cli: Cli) -> Result<()> {
 
     let da_client = match cli.run.da_layer {
         Some(da_layer) => {
-            let da_conf = cli.run.clone().da_conf.unwrap_or({
```

Comment:
The issue was using unwrap_or rather than unwrap_or_else?

Reply:
yes

Reply:
but we can't use unwrap or else in this case so I figured a match would work just as well